Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -1294,6 +1294,8 @@ public void RaiseSpeechCommandRecognized(IMixedRealityInputSource source, Recogn
             // Create input event
             speechEventData.Initialize(source, confidence, phraseDuration, phraseStartTime, command);
 
+            FocusProvider?.OnSpeechKeywordRecognized(speechEventData);
```

Comment:
This approach matches the approach used for OnSourceDetected, above.

Reply:
It's not safe to use `?` on monobehaviors. They do not override the null check properly.

Reply:
Oh interesting, I didn't know that. I simply wrote it this way to stay consistent with OnSourceDetected behavior. We may want to fix this in a separate issue. I'm curious, could you please share a reference explaining why we cannot use ? operator with monobehaviors?

Reply:
>It's not safe to use ? on monobehaviors.

While true, FocusProvider isn't a MonoBehaviour.